frontend/sass/_email.scss

```diff
@@ -0,0 +1,6 @@
+.jf-email-subject {
+  font-size: 1rem;
+  font-style: italic;
+  border-bottom: 2px solid $grey-lighter;
+  padding-bottom: 1rem;
+}
```

Comment:
Yay, this new class is super helpful to have. The whole `p:first-child` selector nonsense was definitely a bit fragile.